CMakeDefinitionsLists.txt

```diff
@@ -1,5 +1,3 @@
-set(BCPDIROPT /home/x86_64-unknown-linux_ol7-gnu/coin-2019-09-19)
-#/Users/legraina/local/Bcp-1.4/build)
-set(BCPDIRDBG home/x86_64-unknown-linux_ol7-gnu/coin-2019-09-19)
-#/Users/legraina/local/Bcp-1.4/build-dbg)
-set(BOOST_DIR /home/x86_64-unknown-linux_ol7-gnu/boost-1.57.0)
+set(BCPDIROPT /Users/legraina/local/Bcp-1.4/build)
+set(BCPDIRDBG /Users/legraina/local/Bcp-1.4/build-dbg)
+#set(BOOST_DIR /home/x86_64-unknown-linux_ol7-gnu/boost-1.57.0)
```

Comment:
Why not require to set those as user environment variables

Reply:
I've added this file in the .gitignore normally. Normally, the local file is not committed. We should perhaps just have a sample file less specific.